src/AggregateEventStatistics.hpp

```diff
@@ -8,41 +8,47 @@
 
 #include "EventStatistics.hpp"
 
-namespace warped {
-
```

Comment:
Forget this file, i was testing with a style tool on c++ file forgot to remove from staging